jarviscli/plugins/screencapture.py

```diff
@@ -0,0 +1,35 @@
+from plugin import plugin, require, alias
+import os
+import sys
+import subprocess
+import platform
+import pyautogui
+
+
+@alias("capture", "screen")
+@plugin('screen_capture')
+def capture_screen(jarvis, s):
+
+    if (platform.system() == 'Linux'):
+        proc = subprocess.Popen(['dpkg',
```

Comment:
Hi!
good thing to handle these kind of exceptions. We should also make the installer aware of scrot / pyautogui.

But dpkg is only for ubuntu or debian. The way intended for these kind of checks is:

```
@require(native='scrot')
@plugin('screen_capture')
```